src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1175,8 +1175,8 @@ private void DynamoViewModelRequestPackagePaths(object sender, EventArgs e)
             var customNodeManager = dynamoViewModel.Model.CustomNodeManager;
             var packageLoader = dynamoViewModel.Model.GetPackageManagerExtension().PackageLoader;
             var viewModel = new PackagePathViewModel(packageLoader, loadPackagesParams, customNodeManager);
-            var view = new PackagePathView(viewModel) { Owner = this };
-            view.ShowDialog();
+            //var view = new PackagePathView(viewModel) { Owner = this };
```

Comment:
I guess I should delete this whole method and the debug menu item for raising the manage node and package path dialog as it's now shown under preferences -